src/services/insurance_service.js

```diff
@@ -56,4 +56,17 @@ export default (easypostClient) =>
 
       return this._retrieve(url);
     }
+
+    /**
+     * Refund an {@link Insurance insurance} record by its ID.
+     * See {@link https://www.easypost.com/docs/api/node#refund-an-insurance EasyPost API Documentation} for more information.
+     * @param {string} id - The ID of the insurance to be refunded.
+     * @returns {Insurance} - The refunded insurance.
+     */
+    static async refund(id) {
+      const url = `beta/insurances/${id}/refund`;
```

Comment:
Shouldn't this be in a separate beta namespace? Or at least noted as beta in the docstring?

Reply:
It will be updated later to use the GA endpoint when it's ready